charts/datadog-operator/values.yaml

```diff
@@ -176,3 +176,7 @@ volumeMounts: []
 #   - name: <VOLUME_NAME>
 #     mountPath: <CONTAINER_PATH>
 #     readOnly: true
+
+# livenessProbe -- Add default livenessProbe settings. HTTP GET is not configurable as it is hardcoded in the Operator.
+livenessProbe:
+  periodSeconds: 10
```

Comment:
I think it would be more user friendly if we listed properties here either with defaults from agent chart or or k8s [docs](https://pkg.go.dev/k8s.io/api@v0.28.4/core/v1#Probe). This will save them searching names they want to override.